### DISCUSSION TITLE
Help. How to Convert DeepXDE model to TensorflowLite model.

### DISCUSSION THREAD

**Discussion by Afiqah-Nur** [lululxvi/deepxde#1744] [Q&A]
Hi. After going through all FAQs and issues, I learned that DeepXDE model can be saved through ckpt files. And I have successfully done that. 
However, Im trying to convert the model to tensorflowlite model. By following the tutorials from here: https://github.com/tensorflow/tensorflow/issues/61645 using the ckpt file saved during training DeepXDE.

When I run the model saved, the result was very inaccurate. Perhaps, the model architecture is not correct? Is there any other way to do this? Or if I were to rebuild the model using deepxde.nn.tensorflow_compat_v1.FNN, how to correctly use the the class in conjunction with TensorFlow's tf.train.Checkpoint? I'm including my code snippets below.

Please help me.
<img width="428" alt="Screenshot 2024-05-21 134604" src="https://github.com/lululxvi/deepxde/assets/109529618/d3c85edc-937e-4a74-aa60-40335fc28a27">
<img width="295" alt="Screenshot 2024-05-21 134535" src="https://github.com/lululxvi/deepxde/assets/109529618/54741294-be21-4183-8b4c-b03719764fed">
<img width="301" alt="Screenshot 2024-05-21 134509" src="https://github.com/lululxvi/deepxde/assets/109529618/d2c03b81-8a8a-46fc-b658-ba49d995d1a3">

**Comment by lululxvi** [comment]
I have no experience with TensorflowLite.
